### DISCUSSION TITLE
Stepper temperatures on Fluidd

### DISCUSSION THREAD

**Discussion by Ameb** [QIDITECH/QIDI_Q1_Pro#67] [Ideas]
The TMC2240 used for X/Y have temperature sensors, which klipper might be able to read: 
https://github.com/Klipper3d/klipper/pull/6210

Would it be possible to add the readings to Fluidd UI?

**Comment by CChen616** [comment]
The new version of fluidd should already support this feature. To upgrade Fluidd, you can follow the fluidd's official instructions.

However, please avoid upgrading moonraker or klipper, as this will cause the printer to malfunction.

After upgrading fluidd, you may notice that the Z-axis controls on the fluidd interface are reversed: the up arrow will move the platform down. (In the qidi version, Fluidd was modified to match the visual effect.)
